src/changelog/buildpacks/_posts/2024-11-19-php-composer-2.8.3.md

```diff
@@ -0,0 +1,9 @@
+---
+modified_at: 2024-11-19 12:00:00
+title: 'PHP - Release of Composer versions 2.8.3'
```

Comment:
Isn't there a version to update in the PHP page for a new Composer version?

Reply:
:face_exhaling: You're obviously right, I forgot it.
Fixed in https://github.com/Scalingo/documentation/pull/2877/commits/057a5214bf38090b821cd4b524b5adafc0c33027